From: Tobias Geerinckx-Rice <me@tobias.gr>
Date: Fri, 29 Nov 2019 20:32:54 +0100
Subject: [PATCH]: gnu: retroarch: Disable the on-line updater.

This disables the entire ‘Online Updater’ sub-menu to address
<http://issues.guix.gnu.org/issue/38360>.  Perhaps that is more than is
necessary.

diff -Naur a/menu/menu_displaylist.c b/menu/menu_displaylist.c
--- a/menu/menu_displaylist.c	1970-01-01 01:00:01.000000000 +0100
+++ b/menu/menu_displaylist.c	2019-11-29 18:35:27.467948854 +0100
@@ -8444,11 +8444,6 @@
                      MENU_ENUM_LABEL_NETPLAY,
                      PARSE_ACTION, false) == 0)
                   count++;
-            if (settings->bools.menu_show_online_updater)
-               if (menu_displaylist_parse_settings_enum(info->list,
-                     MENU_ENUM_LABEL_ONLINE_UPDATER,
-                     PARSE_ACTION, false) == 0)
-                  count++;
             if (menu_displaylist_parse_settings_enum(info->list,
                   MENU_ENUM_LABEL_SETTINGS, PARSE_ACTION, false) == 0)
                count++;
diff -Naur retroarch.a/menu/menu_setting.c retroarch.c/menu/menu_setting.c
--- a/menu/menu_setting.c	1970-01-01 01:00:01.000000000 +0100
+++ b/menu/menu_setting.c	2019-11-29 18:35:35.753957312 +0100
@@ -7291,14 +7291,6 @@
                &group_info,
                &subgroup_info,
                parent_group);
-
-         CONFIG_ACTION(
-               list, list_info,
-               MENU_ENUM_LABEL_ONLINE_UPDATER,
-               MENU_ENUM_LABEL_VALUE_ONLINE_UPDATER,
-               &group_info,
-               &subgroup_info,
-               parent_group);
 #endif
 
          CONFIG_ACTION(
